Integrations/Snowflake/Snowflake.yml

```diff
@@ -0,0 +1,179 @@
+category: Database
+commonfields:
+  id: Snowflake
+  version: -1
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Credentials
+  name: credentials
+  required: true
+  type: 9
+- display: Account
+  name: account
+  required: true
+  type: 0
+- display: Region (only if you are not US West)
+  name: region
+  required: false
+  type: 0
+- display: Authenticator (optional for Okta)
+  name: authenticator
+  required: false
+  type: 0
+- display: Default warehouse to use
+  name: warehouse
+  required: false
+  type: 0
+- display: Default database to use
+  name: database
+  required: false
+  type: 0
+- display: Default schema to use
+  name: schema
+  required: false
+  type: 0
+- display: Default role to use
+  name: role
+  required: false
+  type: 0
+- display: ┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉‎                                                                                  Fetch
+    query to retrieve new incidents
+  name: fetch_query
+  required: false
+  type: 0
+- defaultvalue: 24 hours
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  name: fetch_time
+  required: false
+  type: 0
+- display: The name of the field/column that contains the datetime object or timestamp
+    for the data being fetched (case sensitive)
+  name: datetime_column
+  required: false
+  type: 0
+- display: The name of the field/column in the fetched data from which the name for
+    the demisto incident will be assigned (case sensitive)
+  name: incident_name_column
+  required: false
+  type: 0
+- defaultvalue: 'true'
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+description: Your analytic data warehouse provided as Software-as-a-Service
+display: Snowflake
+name: Snowflake
+script:
+  commands:
+  - arguments:
+    - default: true
+      description: The query to execute
+      isArray: false
+      name: query
+      required: true
+      secret: false
+    - default: false
+      description: Warehouse to use for the query. Will use default if not provided.
```

Comment:
what default ?

Reply:
To address this point I changed the command argument descriptions to be more specific - for this particular command argument the example being 
`Warehouse to use for the query. Will use the warehouse value specified in the integration parameters if it is not provided here.`